src/components/Header/Header.tsx

```diff
@@ -0,0 +1,64 @@
+import { useContext, useState } from 'react';
+import { Todo } from '../../types/todo';
+import { TodosContext } from '../todosContext';
+
+export const Header: React.FC = () => {
+  const { items, setItems } = useContext(TodosContext);
+
+  const [newTodo, setNewTodo] = useState({
+    id: 0,
+    title: '',
+    completed: false,
+  });
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target;
+
+    setNewTodo((prevTodo: Todo) => ({
+      ...prevTodo,
+      [name]: value.replace(/[^a-zA-Zа-яА-Я0-9\s]/g, ''),
+      id: +new Date(),
```

Comment:
Try to use [crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)